apps/src/netsim/NetSimLocalClientNode.js

```diff
@@ -603,6 +609,21 @@ NetSimLocalClientNode.prototype.onMessageTableChange_ = function() {
     // at us.  We must manually call a receive method instead.
     return;
   }
+  if (
+    // messageGranularity === BITS implies this is a simplex wire
+    NetSimGlobals.getLevelConfig().messageGranularity ===
+    MessageGranularity.BITS
+  ) {
+    this.getLatestMessageOnSimplexWire(
+      function(err, message) {
+        if (err) {
+          logger.error('Error pulling message off the wire: ' + err.message);
+          return;
+        }
+        this.wireStateChange.notifyObservers(message.payload);
+      }.bind(this)
```

Comment:
Feel free to use arrow functions in netsim code - they only reason you see `.bind(this)` everywhere is that it was written before our JS transpiler supported ES6.